configs/components/fesom/fesom.yaml

```diff
@@ -16,8 +16,6 @@ available_versions:
 - '2.0'
 - '2.0-r' # OG: temporarily here
 - '2.0-jio'
-- '2.1'
```

Comment:
@vkolatschek , so you know why did you need to delete this line?

Reply:
I don't think I did? I did only change the naming of the restart files. Any other differences must be there, because 'my' fesom.yaml file was not up to date.

Reply:
great, thanks for that info :)